### PR TITLE
add create acc stp1

### PR DESCRIPTION
**Pull request by pivilartisant**
Full screen : 
Screenshot from 2023-05-15 15-01-47

Note : the stepper is not centered but that has been addressed in a ui-kit pr

### CODE REVIEW

File: web-frontend/src/pages/CreateAccount/StepOne.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+import { FiArrowRight } from 'react-icons/fi';
+import LandingPage from '../../layouts/LandingPage/LandingPage';
+
+export default function StepOne() {
+  const defaultFlex = 'flex flex-col justify-center items-center align-center';
+  return (
+    <LandingPage>
+      <div
+        className={`${defaultFlex} 
+                      h-screen`}
```
Comment (mazmassa) [inline web-frontend/src/pages/CreateAccount/StepOne.tsx:13]:
```suggestion
        className="flex flex-col justify-center items-center align-center h-screen"
```

File: web-frontend/src/pages/CreateAccount/StepOne.tsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+          <div className="w-full max-w-xs mb-6">
+            <Stepper step={0} steps={['one', 'two', 'three']} />{' '}
+          </div>
+          <h1 className="mas-banner text-neutral mb-6">Create an account</h1>
+          <div className="w-full mb-4">
+            <Input placeholder={'Enter an account name'} />
+          </div>
+          <div className="w-full">
+            <Button posIcon={<FiArrowRight className="h-6 w-6" />}>
+              {' '}
```
Comment (mazmassa) [inline web-frontend/src/pages/CreateAccount/StepOne.tsx:25]: 🤔

File: web-frontend/src/pages/CreateAccount/StepOne.tsx [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+  const defaultFlex = 'flex flex-col justify-center items-center align-center';
+  return (
+    <LandingPage>
+      <div
+        className={`${defaultFlex} 
+                      h-screen`}
+      >
+        <div className="flex flex-col justify-center content-center items-center w-fit h-fit max-w-lg">
+          <div className="w-full max-w-xs mb-6">
+            <Stepper step={0} steps={['one', 'two', 'three']} />{' '}
```
Comment (Thykof) [inline web-frontend/src/pages/CreateAccount/StepOne.tsx:17]:
```suggestion
            <Stepper step={0} steps={['Username', 'Password', 'Back-up']} />{' '}
```

**Review by Thykof — CHANGES_REQUESTED** [review]

File: web-frontend/src/pages/CreateAccount/StepOne.tsx (hunk 2 above)
Comment (pivilartisant) [reply web-frontend/src/pages/CreateAccount/StepOne.tsx]: changed

File: web-frontend/src/pages/CreateAccount/StepOne.tsx (hunk 1 above)
Comment (pivilartisant) [reply web-frontend/src/pages/CreateAccount/StepOne.tsx]: changed

File: web-frontend/src/pages/CreateAccount/StepOne.tsx (hunk 3 above)
Comment (pivilartisant) [reply web-frontend/src/pages/CreateAccount/StepOne.tsx]: changed !!

File: web-frontend/src/pages/CreateAccount/StepOne.tsx [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+import { FiArrowRight } from 'react-icons/fi';
+import LandingPage from '../../layouts/LandingPage/LandingPage';
+
+export default function StepOne() {
+  return (
+    <LandingPage>
+      <div
+        className={`flex flex-col justify-center items-center align-center
+                      h-screen`}
+      >
```
Comment (Thykof) [inline web-frontend/src/pages/CreateAccount/StepOne.tsx:13]: why prettier let this div on 4 lines?

```suggestion
      <div className={`flex flex-col justify-center items-center align-center h-screen`}>
```

**Review by Thykof — APPROVED** [review]

File: web-frontend/src/pages/CreateAccount/StepOne.tsx (hunk 4 above)
Comment (pivilartisant) [reply web-frontend/src/pages/CreateAccount/StepOne.tsx:13]: i have no idea I'll make the change :/

File: web-frontend/src/main.tsx [hunk 5]
```diff
@@ -22,6 +23,7 @@ const router = createBrowserRouter(
       <Route path="index" element={<Welcome />} />
       <Route path="account-select" element={<SelectAccount />} />
       <Route path="account-new" element={<AddAccount />} />
+      <Route path="account-create-step1" element={<StepOne />} />
```
Comment (mazmassa) [inline web-frontend/src/main.tsx:26]:
```suggestion
      <Route path="account-create-step-one" element={<StepOne />} />
```

File: web-frontend/src/pages/CreateAccount/StepOne.tsx [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+
+export default function StepOne() {
+  return (
+    <LandingPage>
+      <div
+        className={`flex flex-col justify-center items-center align-center h-screen`}
+      >
+        <div className="flex flex-col justify-center content-center items-center w-fit h-fit max-w-sm">
+          <div className="w-full max-w-xs mb-6">
+            <Stepper step={0} steps={['Username', 'Password', 'Back-up']} />{' '}
```
Comment (mazmassa) [inline web-frontend/src/pages/CreateAccount/StepOne.tsx:15]: Backup in one word: **Backup**

**Review by mazmassa — APPROVED** [review]
Left some comments to fix, but overall LGTM!

**Pull request merged**